content/storage/postgres/docker-compose-for-postgres.md

```diff
@@ -5,14 +5,14 @@ editable: true
 ---
 
 To use Postgres Docker compose, we need to create required files and add contents to theme.
-- main docker-compose.yml
+- main docker-compose.[yml/yaml]
```

Comment:
Is it rarely for people to use yaml?

Reply:
I'm not sure even me use yaml

Reply:
@dmakeroam The shorter, the better bro.
However, I still use .html even though I think I can use .htm. LoL.
Therefore, I think it is good to have them both in our example.
Thanks.

Reply:
NC